cypress/integration/advanced-functionalities/server/mfa/Disable.spec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+const adminURL = Cypress.env('ADMIN_URL')
+const email = Cypress.env('USER_EMAIL_NEW')
+const password = Cypress.env('USER_PASSWORD_NEW')
+
+describe('Test for disabling multi-factor authentication', () => {
+  before(() => {
+    if (!window.sessionStorage.getItem('auth.refresh-token')) {
+      cy.login({ email, password, url: adminURL })
+    }
+  })
+
+  context('Test for disabling multi-factor authentication', () => {
+    it('should be able to disable multi-factor authentication', () => {
+      cy.intercept('/api/system/stats/').as('load')
+      cy.intercept('/api/system/settings/').as('auth-settings')
+      cy.visit(adminURL + '/')
+      cy.wait('@load')
+      cy.get('.nav-sidebar').contains('Auth Settings').click({ force: true })
```

Comment:
`contains` can bring a lot of work, since if the translations will change, we will have to fix the tests as well. just fyi, do what you want with it :) 
The rule is, test with data or meta attributes, not content and design